test/support/livebook_case.ex

```diff
@@ -1,6 +1,12 @@
 defmodule Kino.LivebookCase do
   use ExUnit.CaseTemplate
 
+  using do
+    quote do
+      import KinoTest.Livebook
```

Comment:
FWIW, you can have the helper functions in this module and then import this module itself.